src/main/resources/assets/memoryusagescreen/lang/ru_ru.json

```
@@ -1,8 +1,8 @@
 {
-  "text.autoconfig.memoryusagescreen.title": "Настройки Memory Usage Screen",
-  "text.autoconfig.memoryusagescreen.option.enableInitLoadingScreen": "Включить отображение на экране загрузки игры",
-  "text.autoconfig.memoryusagescreen.option.enableWorldLoadingScreen": "Включить отображение на экране загрузки мира",
-  "text.autoconfig.memoryusagescreen.option.enableToggleMode": "Режим переключения отображения по нажатию",
+  "text.autoconfig.memoryusagescreen.title": "Конфигурация Memory Usage Screen",
+  "text.autoconfig.memoryusagescreen.option.enableInitLoadingScreen": "Включить отображение на начальном загрузочном экране.",
```

Просто не понимаю, зачем усложнять формулировку непрямой структурой построения связей в предложении. Это тот случай, когда незначительное отхождение от сырого перевода значительно улучшает восприятие.
```suggestion
  "text.autoconfig.memoryusagescreen.option.enableInitLoadingScreen": "Включить отображение на экране загрузки игры",
```

---

Во-первых, не усложнял формулировку, а написал как должно быть написано изначально, а не придумывать и добавлять лишние слова. А если человек не понимает значение параметра, то есть к этому описание, никто не виноват, если сам разработчик не добавляет описание к параметрам.